src/Providers/ConversionServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Conversion\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Conversion\Converter;
+use Conversion\Parser;
+use Conversion\Registry;
+use Conversion\RegistryBuilder;
+
+class ConversionServiceProvider extends ServiceProvider
+{
+    public function register()
+    {
+
+        $this->app->singleton('converter', function ($app) {
```

Comment:
```suggestion
        $this->app->singleton(Converter::class, function (Application $app) {
```
We should use the class name instead of a string to avoid conflicts. This will also allow injection in commands, controllers, etc.